internal/fbsemantic/semantic.fbs

```diff
@@ -255,6 +261,12 @@ table ArrayExpression {
   typ:MonoType;
 }
 
+table VectorExpression {
+  loc:SourceLocation;
+  elements:[WrappedExpression];
+  typ:MonoType;
+}
+
```

Comment:
Again here as well, there is no need to create a `VectorExpression` node, same reason as the comment about AST.